components/scg/src/scg-struct.js

```diff
@@ -234,6 +234,31 @@ function scgScStructTranslator(_editor, _sandbox) {
             /// --------------------
             var translateNodes = async function () {
                 var implFunc = async function (node) {
+                    if (node.text) {
```

Comment:
Probably this code should go later inside` if (!node.sc_addr)` block

Reply:
Why? We get sc_addr from system identifier